drivers/include/cc1xxx_common.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *               2018 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_cc1xxx CC1100/CC1100e/CC1101/CC1200 common code
+ * @ingroup     drivers_netdev
+ * @ingroup     net_gnrc_netif
+ * @{
+ *
+ * @file
+ * @brief   CC110x/CC1200 adaption for @ref net_gnrc_netif
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ * @author  Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author  Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ *
+ * Supported Transceivers
+ * ======================
+ *
+ * This adaption layer is written to be used by CC110x and CC1200 transceivers,
+ * but any transceiver using that should transfer 6LoWPAN and uses layer 2
```

Comment:
Any payload fitting into the MTU of 255 Bytes minus the header (2 bytes currently). However, currently layer 2 does not have a field to identify the protocol of the frames payload, which makes using multiple protocols simultaneously difficult. I plan to extend the layer 2 format in later PRs to address this.

Still, the this code would allow even yet to overwrite `C1XXX_DEFAULT_PROTOCOL` to use carry other protocols. So indeed, that commend is misleading.

Reply:
> However, currently layer 2 does not have a field to identify the protocol of the frames payload, which makes using multiple protocols simultaneously difficult. I plan to extend the layer 2 format in later PRs to address this.
>
> Still, the this code would allow even yet to overwrite C1XXX_DEFAULT_PROTOCOL to use carry other protocols. So indeed, that commend is misleading.

Yepp, especially if you try using GNRC with out the 6LoWPAN module with this driver ;-) (which AFAIK `examples/default` does.